### PR TITLE
Headless Integration

### PR DESCRIPTION
**Pull request by digvijay-otpless**

### CODE REVIEW

File: example/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -28,7 +29,7 @@
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data
                     android:host="otpless"
-                    android:scheme= "otpless.your_app_id_in_lowercase"/>
+                    android:scheme= "otpless.5e62zcanetd9urnxpz80"/>
```
Comment (digvijay-otpless) [inline example/android/app/src/main/AndroidManifest.xml]: remove

File: example/ios/App/App/Info.plist [hunk 2]
```diff
@@ -7,7 +7,7 @@
       <dict>
        <key>CFBundleURLSchemes</key>
        <array>
-        <string>otpless.your_app_id_in_lowercase</string>
+        <string>otpless.5e62zcanetd9urnxpz80</string>
```
Comment (digvijay-otpless) [inline example/ios/App/App/Info.plist]: remove

File: example/src/pages/Home.tsx [hunk 3]
```diff
@@ -1,51 +1,142 @@
...
   let manager = new OtplessManager()
-  const [result, setResult] = useState("Result:")
-  OtplessInstance.removeAllListeners();
-  OtplessInstance.addListener('OtplessResultEvent', (result: any) => {
-    console.log("data in listener: " + result);
-    handleResult(result);
-    });
+
+  useEffect(() => {
+    manager.initHeadless("5E62ZCANETD9URNXPZ80")
```
Comment (digvijay-otpless) [inline example/src/pages/Home.tsx]: remove

File: example/src/pages/Home.tsx [hunk 4]
```diff
@@ -1,51 +1,142 @@
...
+      ...prevForm, // Keep existing fields
+      [fieldName]: value, // Update the specific field
+    }));
+  }
 
   var loaderVisibility = true;
 
   const openLoginPage = async() => {
-    let jsonParams = {appId: "APP_ID"}
+    let jsonParams = {appId: "5E62ZCANETD9URNXPZ80"}
```
Comment (digvijay-otpless) [inline example/src/pages/Home.tsx]: remove

**Review by dev-otpless-tech — APPROVED** [review]

**Pull request merged**